src/lib.rs

```diff
@@ -100,6 +100,8 @@ pub use book::BookItem;
 pub use renderer::Renderer;
 
 /// The error types used through out this crate.
+// TODO: Drop after error_chain is fixed
+#[allow(unused_doc_comment)]
```

Comment:
This should probably be pulled out into its own PR. `error-chain` has actually fixed this, so bumping the dependency version to `0.11.0-rc.2` would be sufficient.

Reply:
yeah; I did this on one of my recent projects